src/framework/stubs/CMakeLists.txt

```diff
@@ -30,6 +30,10 @@ if (NOT MUSE_MODULE_CLOUD)
     add_subdirectory(cloud)
 endif()
 
+if (NOT MUSE_MODULE_EXTENSIONS)
+    add_subdirectory(extensions)
```

Comment:
Just a thought: perhaps we can get rid of `framework/stubs/CMakeLists.txt`, and add a macro in `framework/CMakeLists.txt` that either calls `add_subdirectory(extensions)` or `add_subdirectory(stubs/extensions)` based on `MUSE_MODULE_EXTENSIONS`. But that's also something for a separate PR of course.